gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 kotlin = "1.8.10"
 jetbrains-annotations = "24.0.0"
-save-core = "0.3.8"
+save-core = "0.3.9"
```

Comment:
I thought that latest release was 0.3.8

Reply:
We've released version **0.3.9** which now relies on the released (**0.2.1**) version of sarif-utils: saveourtool/save-cli#507.

Reply:
As a side-note, this change aligns `kotlinx.serialization` version (now **1.5.0**) with both `save-core` and `sarif-utils`, which may fix #1924.